addons/cert.py

```diff
@@ -90,7 +90,10 @@ def getArgumentsRecursive(tok, arguments):
         getArgumentsRecursive(tok.astOperand1, arguments)
         getArgumentsRecursive(tok.astOperand2, arguments)
     else:
-        arguments.append(tok)
+        if tok.isArithmeticalOp:
```

Comment:
This does not look right. For such code:

     dostuff(x+2, y*4)

There are 2 arguments; `+` and `*`